src/loguru_parallel/listen.py

```diff
@@ -51,15 +52,30 @@ def stop(self) -> None:
         if self._thread is None:
             return
         super().stop()
-
-    def _monitor(self) -> None:
-        """Monitor the queue for records, and ask the handler to deal with them."""
-        super()._monitor()
         self._sink_logger.complete()
         print("Loguru Listener stopped.")
```

Comment:
@ykfjmt I think this is what you meant. Yes actually we can run this in the `stop` method.